.github/workflows/nightly-upstream-snapshot-build.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 env:
   AWS_DEFAULT_REGION: us-east-1
-  IMAGE_TAG: 611364707713.dkr.ecr.us-west-2.amazonaws.com/adot-autoinstrumentation-java-nightly:nightly
+  IMAGE_NAME: 611364707713.dkr.ecr.us-west-2.amazonaws.com/adot-autoinstrumentation-java-nightly
```

Comment:
Are we removing the nightly tag for the image, and use the timestamp for it.

Reply:
yes